packages/@tests/smoke/web/public/index.js

```diff
@@ -1,11 +1,11 @@
-const peer = globalThis.defaultPeer;
+const fluence = globalThis.fluence;
 
 const relay = {
-    multiaddr: '/ip4/127.0.0.1/tcp/4310/ws/p2p/12D3KooWKEprYXUXqoV5xSBeyqrWLpQLLH4PXfvVkDJtmcqmh5V3',
-    peerId: '12D3KooWKEprYXUXqoV5xSBeyqrWLpQLLH4PXfvVkDJtmcqmh5V3',
+    multiaddr: '/dns4/kras-01.fluence.dev/tcp/19001/wss/p2p/12D3KooWKnEqMfYo9zvfHmqTLpLdiHXPe4SVqUWcWHDJdFGrSmcA',
```

Comment:
seems like this should not be kras

Reply:
The reason I've switched to kras is that smoke test uses calls to registry and they fail on a single local node. At least that's true for test used in CRA and Node.js.

I will add a comment about that

Reply:
maybe to ask @nahsi to set up multiple local nodes like we have it everywhere else?

Reply:
Sounds like a plan. I suggest to live with kras for a little bit and figure out the best way to solve it though!